spec/features/topic_user_feature_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature "Topic User Process:", js: true do
+
+  before do
+    user = create(:user)
+    visit root_path
+    click_link "Login"
+    fill_in :user_email, with: user.email
+    fill_in :user_password, with: user.password
+    click_button "Sign in"
+    expect(current_path).to eq('/')
+    expect(page).to have_content("Signed in successfully.")
+  end
+
+  scenario "user views topic index page" do
+    topic = create(:topic)
+    visit topics_path
+
+    expect(page).to have_content("First Topic")
+    expect(page).to_not have_content("New Topic")
```

Comment:
I think I realize now what this is testing. Its testing the 'New Topic' link isn't on the page. Pleased to see this being tested. 

Consider clarifying this expectation by making it clear its testing for a link:

```diff
-expect(page).to_not have_content("New Topic")
+expect(page).to_not have_link("New Topic")